package.json

```diff
@@ -20,12 +20,13 @@
   "homepage": "https://github.com/broccolijs/broccoli",
   "dependencies": {
     "broccoli-kitchen-sink-helpers": "^0.2.5",
-    "broccoli-slow-trees": "^1.0.0",
+    "broccoli-slow-trees": "^3.0.0",
     "commander": "^2.5.0",
     "connect": "^3.3.3",
     "copy-dereference": "^1.0.0",
     "findup-sync": "^0.2.1",
     "handlebars": "^4.0.4",
+    "heimdalljs": "^0.1.0",
```

Comment:
has that been published yet? https://www.npmjs.com/search?q=heimdalljs looks pretty empty